blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/RollupRunnableIntegrationTest.java

```diff
@@ -95,7 +97,11 @@ public void setUp() throws Exception {
             BluefloodSetRollup rollup = new BluefloodSetRollup().withObject(i);
             metric = new PreaggregatedMetric(time, setLocator, ttl, rollup);
             preaggregatedMetrics.add(metric);
-            
+
```

Comment:
You should move the hashed value calculation here to a separate that can be repeated call for other tests that you're going to be writing ;)